### ISSUE TITLE
Carnitine shuttle  (R-HSA-200425)

### ISSUE THREAD

**Issue by ukemi** [geneontology/pathways2GO#334]
- [ ]  Make the mouse model     http://noctua.geneontology.org/editor/graph/gomodel:675b862c00000923?

**Comment by ukemi** [comment]
@deustp01 

I have split out the reactions in the top right hand side and made them separate regulatory processes where the output small molecules are the regulators of the core enzyme of the shuttle.

**Comment by ukemi** [comment]
I've also split the import of carnitine out into its own transport process.

**Comment by ukemi** [comment]
Doesn't PMID:24277613 show that Spot14/Mig12 inhibits the activity of ACACB? Is the direction of regulation incorrect in the pathway?

**Comment by deustp01** [comment]
> Doesn't PMID:24277613 show that Spot14/Mig12 inhibits the activity of ACACB? Is the direction of regulation incorrect in the pathway?

GitHub oddity: I also got a GitHub note that this comment was also attached to ticket #333 (ubiquinol biosynthesis) but I can't find it there. I also can't find any role for ACACB in that pathway, so this looks like some sort of GitHub AI hallucination.

Meanwhile, I can find a role for ACACB in the carnitine shuttle (this ticket) and its alleged regulation by the human equivalents of Spot14/Mig12, so I will get the paper you cite and sort this out along with the pathway boundary issues above.

**Comment by ukemi** [comment]
Based on the blurb for R-HSA-200421, does it make more sense to have the AMP regulate either the dephosphorylation in R-HSA-380949 or as an activator of the AMPK activity in R-HSA-380930? Note this reaction gets into the GO-CAM carnitine shuttle as an upstream reaction.

I had originally made the other comment on the wrong ticket and then deleted it and added it to this one. Too many GH issues open at once. It goes with this one.

**Comment by ukemi** [comment]
The regulation of these lipid pathways is wild!

**Comment by deustp01** [comment]
Regulation is wild.

We have probably lumped too many distinct processes together here. Going back to old med student lectures, they remind us that malonyl CoA is the key starting point for synthesis of long-chain fatty acids, a cytosolic process involving malonyl CoA synthesis by ACACA. Meanwhile, fatty acid catabolism to acetyl CoA to generate energy proceeds inside the mitochondrion. Those fatty acids get there from the cytosol by carnitine-mediated transport.

In the liver, that transport is negatively regulated by malonyl CoA (made by ACACA), thereby preventing a futile cycle in which freshly-made cytosolic fatty acids get shipped to the mitochondrion and broken down.

In other tissues, ACACB makes smaller amounts of malonyl CoA, which regulates mitochondrial uptake of fatty acids.

So regulation of ACACA and ACACB should be different, possibly reciprocal, an issue that we have appeared to have overlooked in our carnitine shuttle annotations, that PMID:24277613 forces us to notice.
<img width="562" alt="old_dwg_board" src="https://github.com/user-attachments/assets/4a343563-0808-473c-9ac6-bc8796c58adb" />

**Comment by deustp01** [comment]
Note to self with some human UniProt IDs that will be needed for the cleanup - all have bibliographies that look useful
[MID1IP1](https://www.uniprot.org/uniprotkb/Q9NPA3/entry)
[ACACA](https://www.uniprot.org/uniprotkb/Q13085/entry)
[ACACB](https://www.uniprot.org/uniprotkb/O00763/entry)

**Comment by ukemi** [comment]
> We have probably lumped too many distinct processes together here.

Yes, I am concerned about this in the GO-CAM models as well. A while ago it was decided on a call that we would include regulation in the core model of the process. The problem with that.... regulation is very context-specific and lumping all they types of regulation for a process like this into a single model could be very misleading.

**Comment by ukemi** [comment]
After reviewing your slides, which were fantastic, I think the GO-CAM group should reconsider the lumping of regulatory processes. It would be really nice to be able to actually link models.

**Comment by deustp01** [comment]
There may also be a lumping issue at the level of the metabolic process itself. Reactome uses sets of protein isoforms to provide compact representations of single-step reactions where the identical chemical transformation is enabled by more than one gene product.

The rationale for this strategy is that it supports human-friendly visualization of pathways - one string of reactions versus many parallel ones, and that it makes annotation and maintenance easier and maybe more reliable. The result represents all tissue- and state-specific versions of a pathway and its regulation as a single process. Users can dissect out the version of the pathway relevant to a specific tissue and physiological state by overlaying tissue- and state-specific expression data. That overlay should be equally effective at picking out the tissue- and state-specific regulatory features of the process.

The issue now is whether that strategy really works in practice. If not, then in addition to thinking about whether to annotate process and regulation-of-process separately and tie them together with between-process links (do-able now in the Reactome process description world; do-able in principle in the GO-CAM activity flow world though more work is needed to implement it in a robust way), we may need to think about whether we need to dissect those tissue and state-specific forms of processes apart, and then decorate each of those specific pathway / process annotations with its appropriate specific regulatory features.

**Comment by ukemi** [comment]
>  isn't this another way of saying that fiddling with pathway boundaries will affect expression analyses, so we have just tripped over a new instance of an old standard hard problem?

Absolutely! And depending on the type of study, you may want the entire 'bag of genes' and even more. I'm thinking along the lines of the phenotype analyses we did for our last paper.

**Comment by ukemi** [comment]
I think at the end of the day, what we really want is the ability to operate dynamically across all the data. Although a model/pathway is a convenient way to limit groupings, the ability to query across or within groupings (both sets and pathways/models) with other limits like the expression conditions you mention will be a more complete picture of the biology.

**Comment by deustp01** [comment]
> In the liver, that transport is negatively regulated by malonyl CoA (made by ACACA), thereby preventing a futile cycle in which freshly-made cytosolic fatty acids get shipped to the mitochondrion and broken down.
> 
> In other tissues, ACACB makes smaller amounts of malonyl CoA, which regulates mitochondrial uptake of fatty acids.

This is wrong. Wakil, who discovered all of this, explains ([PMID: 19047759](https://pubmed.ncbi.nlm.nih.gov/19047759/)):

> While ACC1 [ACACA] is generally expressed in all tissues, it is expressed more in lipogenic tissues: liver, adipose, and lactating mammary gland. In contrast, ACC2 [ACACB] (Mr 282,000) is highly expressed in heart and muscle and to a lesser extent in liver ([4](https://pmc.ncbi.nlm.nih.gov/articles/PMC2674721/#bib4)). ... The ACC1-generated malonyl-CoA is utilized by FAS for the synthesis of fatty acids in the cytosol. In contrast, the ACC2-generated malonyl-CoA functions as inhibitor of the carnitine/palmitoyl-transferase 1 (CPT1) activity and the transfer of the fatty acyl group through the carnitine/palmitoyl shuttle system to inside the mitochondria for β-oxidation ([Fig. 1](https://pmc.ncbi.nlm.nih.gov/articles/PMC2674721/#fig1)). The malonyl-CoA generated by ACC1 and ACC2 within the cell do not mix and are highly segregated.

Adding to the confusing, we have two essentially identical ACACA-enabled malonyl-CoA synthesis reactions, one that is part of long-chain fatty acid biosynthesis ([R-HSA-200555](https://reactome.org/PathwayBrowser/#/R-HSA-75105&SEL=R-HSA-200555&PATH=R-HSA-1430728,R-HSA-556833,R-HSA-8978868&FLG=UniProt:Q13085) Formation of Malonyl-CoA from Acetyl-CoA (liver)) and one that is part of the carnitine shuttle ([R-HSA-75851](https://reactome.org/PathwayBrowser/#/R-HSA-200425&SEL=R-HSA-75851&PATH=R-HSA-1430728,R-HSA-556833,R-HSA-8978868&FLG=UniProt:Q13085) Btn-ACACA:2Mn2+ polymer carboxylates Ac-CoA to form Mal-CoA). This second one, as as well as being a duplicate, is dead wrong per Wakil. And to the extent, if any, that Spot14/Mig12 regulates ACACA that regulation should be attached to the fatty acid biosynthesis version of the reaction.

**Comment by deustp01** [comment]
> Adding to the confusion,

@ukemi In fact, we already tripped over this problem in August when we were revising the carnitine pathways as part of the molecular events cleanup and realized that these reactions were mis-grouped but didn't fully fix things - see notes for reactions 75851, 8876889, and 200423 in the [carnitine notes GoogleDoc](https://docs.google.com/document/d/1qmtjiFs650flANn8y0I8k5UDL0syUG6trIC4GzcjAn4/edit?tab=t.0).
